app/customHooks/useDigitConvert.ts

```diff
@@ -0,0 +1,68 @@
+import { retry } from '@redux-saga/core/effects';
+import { useState } from 'react';
+import { useAppSelector } from '../../App';
+
+// Define type for supported languages and their digit mappings
+type DigitLanguage = 'bn' | 'hin' | 'urd' | 'ar' | 'tr' | 'en';
+
+const useDigitConverter = (): {
+    language: DigitLanguage;
+    setLanguage: React.Dispatch<React.SetStateAction<DigitLanguage>>;
+    convertDigits: (inputString: any) => string;
+} => {
+
+    const locale = useAppSelector((state: any) =>
+    state.selectedCountry?.locale,
+  );
+    // Define digit mappings for different languages
+    const digitMap: Record<DigitLanguage, string[]> = {
+        'bn': ['০', '১', '২', '৩', '৪', '৫', '৬', '৭', '৮', '৯'],
+        'hin': ['०', '१', '२', '३', '४', '५', '६', '७', '८', '९'],
+        'urd': ['۰', '۱', '۲', '۳', '۴', '۵', '۶', '۷', '۸', '۹'],
+        'ar': ['٠', '١', '٢', '٣', '٤', '٥', '٦', '٧', '٨', '٩'],
+        'tr': ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'],
+        'en': ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'], // Turkish uses Latin digits
+        // Add more languages and their digit mappings as needed
+    };
+
+    const [language, setLanguage] = useState<DigitLanguage>('bn'); // Default language is Bangla
```

Comment:
Is default language Bengali, otherwise we need to correct or remove the comment.